### DISCUSSION TITLE
성격이 급해서 Excel로 해버린 시기 별 EDA (월, 주, 일, 요일, 시간)

### DISCUSSION THREAD

**Discussion by sooyounlee** [bcaitech1/p4-dkt-DKTJHGSD#60] [Show and tell]
**결론** :
날짜 중에서는 주/요일을 사용하면 적절해보입니다. 일요일에 낮아지고, 11-12월이 포함된 주에 낮아지는 패턴을 보이는 것 같아요. 이것도 숫자 그대로 넣고 카테고리로 나눠서 일요일/나머지, 기말/나머지 이렇게 넣어도 좋을 것 같네요. 시간은 그대로 넣어줘도 좋을 것 같아요.

- 월별 정답률: 11, 12월이 현격히 낮습니다. 
![image](https://user-images.githubusercontent.com/21062594/120003260-8ba83b00-c010-11eb-8eaa-09fa2a1e1808.png)

- 주별 정답률: 주별로 정답률을 보니 점차 하락하네요. 이렇게 되면 월별이 아니라 주별로 봐야할까요? 일별 정답률 그래프도 보도록 하겠습니다.
![image](https://user-images.githubusercontent.com/21062594/120004147-68ca5680-c011-11eb-98f7-b3b519bb278f.png)

- 일별 정답률: 일별 정답률도 뭔가 패턴이 있어보이긴 하는데, 요일별 정답률과 주별 정답률의 영향인 것 같아요. 주와 요일을 넣는게 적절해보이는 것 같습니다.
![image](https://user-images.githubusercontent.com/21062594/120008271-c19bee00-c015-11eb-977a-b7a366f45f5c.png)

- 요일 별 정답률: 일요일이 아주 약간 낮네요.
![image](https://user-images.githubusercontent.com/21062594/120003321-995dc080-c010-11eb-9a27-0b1c9e027e0d.png)

- 시간대 별 정답률: 오후 4시에 바닥이었다가 서서히 올라가는 패턴을 보이는 것 같아요. 시간만 따로 빼서 feature로 사용해도 좋을 것 같습니다.
![image](https://user-images.githubusercontent.com/21062594/120003929-29036f00-c011-11eb-93aa-907c0fc89e1f.png)

**Comment by junho621** [comment]
오오오 아주 양질의 eda 같습니다!! 사용해보도록 할게요!!

**Reply by junho621** [reply]
수연님 기말이 뭔지 알려주실 수 있나요 ?

**Reply by sooyounlee** [reply]
기말고사(final test)의 기말을 쓴건데, 이렇게 기말이라고 생각해보니 11월 12월에 더 어려운 문제지를 푸는 경향이 있을 수도 있을 것 같아요. 저 기말 feature는 월별 혹은 주별로 푸는 문제지가 달라지는지를 먼저 보는게 좋을 것 같아요! 11월 12월에 학년 마무리하면서 더 어려운 시험들을 볼 수도 있으니까요.. 그럼 월이나 주보다는 시험지 난이도로 변수를 만들어서 넣는게 더 직접적일 것 같아요! 

혹시 먼저 주 별을 넣어서 실험해보고 싶으시면 넣으셔도 좋아요 ㅋㅋ 사실 주나 월이나 자르는게 크게 달라지지 않을 것 같아서 월별로 그룹 만들어서 11월 12월을 final이라고 둬도 괜찮을 것 같네요.. 
근데 일단은 요일이랑 시간 변수만 먼저 넣어보는게 좋을 것 같아요!

**Comment by sooyounlee** [comment]
다음 학년거 예습 시작한거 아님? (-태환)

**Reply by sooyounlee** [reply]
예습은 방학때 하는거 아닌가요? (-서윤)
